weave

```diff
@@ -74,9 +74,15 @@ EOF
 }
 
 exec_remote() {
+    HOSTS_PATH=$(docker $DOCKER_CLIENT_ARGS run --rm --net=none \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        --entrypoint /bin/sh \
+        $EXEC_IMAGE -c 'docker inspect -f {{.HostsPath}} $(hostname)')
```

Comment:
If you're modifying `/etc/hosts` in the weave script, won't that also need this? Either way, doing it only for `attach` and/or `launch-proxy` seems like a good idea.